src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/VideoBooking.kt

```diff
@@ -178,8 +178,8 @@ class VideoBooking private constructor(
       probationMeetingType = probationMeetingType,
       comments = comments,
       videoUrl = null,
-      createdBy = createdBy,
-      createdByPrison = createdByPrison,
+      createdBy = createdBy.username,
+      createdByPrison = createdBy is PrisonUser,
```

Comment:
Pushed this down into the entity.  Will do the same for court bookings in a separate PR.